robot_description_param: "/robot_description" 
joint_states_topic:  "/joint_states"
base_frame: "base_link"
ee_frame: "ee_link"
frequency: 10.0
sensors: [-1, 3, 2, 2, 2, 3, 3] #-1: None; 0: FTsensor; 1: Force_x; 4:Torque_x, ...